ludwig/api.py

```diff
@@ -661,8 +661,8 @@ def on_epoch_end(self, trainer, progress_tracker, save_path):
                                 calibrator.train_calibration(training_set, TRAINING)
                             else:
                                 calibrator.train_calibration(validation_set, VALIDATION)
-                        if not skip_save_model:
-                            self.model.save(model_dir)
+                        # if not skip_save_model:
+                        # self.model.save(model_dir)
```

Comment:
@arnavgarg1 I think you mentioned that you had a fix in mind for this– is there something long-term we can do here?

Reply:
Yeah, I'll have something in as a part of the prompt-tuning PR. 
Essentially, we will only save model checkpoints if there is an `adapter` being used for fine-tuning, same with loading. Otherwise, it should just do the right thing with just the `model_name` being provided afaik